examples/imagenet_featadvs/attack_model_featadv.py

```diff
@@ -0,0 +1,160 @@
+"""Runs CleverHans attacks on the Madry Lab MNIST challenge model
```

Comment:
If this is for the MNIST challenge, why is it in a directory with "imagenet" in the name?

Reply:
Fixed in 47b04aa.